### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by geokar2006**

### CODE REVIEW

File: Extensions/dnSpy.Debugger/dnSpy.Debugger/Properties/dnSpy.Debugger.Resources.ru.resx [hunk 1]
```diff
@@ -1530,4 +1530,34 @@
...
+  </data>
+  <data name="StartDebuggingWarning_IncorrectBitness" xml:space="preserve">
+    <value>Разрядность целевого файла не соответствует текущей версии dnSpy. Отладка может привести к нежелательному/неконтролируемому выполнению образа.
+
+Рекомендуется использовать 32-разрядный dnSpy для отладки 32-разрядных образов и 64-разрядный dnSpy для отладки 64-разрядных образов.
+
+Вы хотите продолжить?</value>
+  </data>
+  <data name="DbgSettings_EnableJustMyCodeDebugging" xml:space="preserve">
+    <value>Включите поддержку отладки только моего кода</value>
```
Comment (MadProbe) [inline Extensions/dnSpy.Debugger/dnSpy.Debugger/Properties/dnSpy.Debugger.Resources.ru.resx:1555]: Would it be better if this was "Включите поддержку отладки только для моего кода"?

File: Extensions/ILSpy.Decompiler/dnSpy.Decompiler.ILSpy.Core/Properties/dnSpy.Decompiler.ILSpy.Core.Resources.ru.resx [hunk 2]
```diff
@@ -234,4 +234,10 @@
   <data name="DecompilerSettings_UseSourceCodeOrder" xml:space="preserve">
     <value>Сортировать члены как в исходном коде</value>
   </data>
-</root>
+  <data name="DecompilerSettings_EmitCalliAsInvocationExpression" xml:space="preserve">
+    <value>Декомпилировать calli в выражение вызова</value>
```
Comment (MadProbe) [inline Extensions/ILSpy.Decompiler/dnSpy.Decompiler.ILSpy.Core/Properties/dnSpy.Decompiler.ILSpy.Core.Resources.ru.resx:238]: Do you think that "Декомпилировать calli в виде выражения вызова" would be better here?
This is good enough anyway though.

**Review by MadProbe — APPROVED** [review]
I have done through this PR and as a Russian I generally approve these changes.
My comments are purely editorial and can be discarded in favor of compressing sentences so text fits on screen better, if applicable.

File: Extensions/dnSpy.Analyzer/Properties/dnSpy.Analyzer.Resources.ru.resx [hunk 3]
```diff
@@ -210,4 +210,4 @@
   <data name="UsesTreeNode" xml:space="preserve">
     <value>Использует</value>
   </data>
-</root>
+</root>
```
Comment (ElektroKill) [inline Extensions/dnSpy.Analyzer/Properties/dnSpy.Analyzer.Resources.ru.resx:213]: Undo changes to the file endings too (in many other files!)

File: Extensions/ILSpy.Decompiler/dnSpy.Decompiler.ILSpy.Core/Properties/dnSpy.Decompiler.ILSpy.Core.Resources.ru.resx [hunk 4]
```diff
@@ -60,45 +60,45 @@
             : and then encoded with base64 encoding.
     -->
   <xsd:schema xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata" id="root">
-    <xsd:import namespace="http://www.w3.org/XML/1998/namespace"/>
+    <xsd:import namespace="http://www.w3.org/XML/1998/namespace" />
```
Comment (ElektroKill) [inline Extensions/ILSpy.Decompiler/dnSpy.Decompiler.ILSpy.Core/Properties/dnSpy.Decompiler.ILSpy.Core.Resources.ru.resx:63]: Please undo whitespace changes in all the files (in other files too)

**Review by ElektroKill — CHANGES_REQUESTED** [review]

File: Extensions/dnSpy.Debugger/dnSpy.Debugger/Properties/dnSpy.Debugger.Resources.ru.resx (hunk 1 above)
Comment (geokar2006) [reply Extensions/dnSpy.Debugger/dnSpy.Debugger/Properties/dnSpy.Debugger.Resources.ru.resx:1555]: First translation was rough and through a translator. I forgot to fix translation after translator. I will change to "Включить поддержку отладки только моего кода".

File: Extensions/ILSpy.Decompiler/dnSpy.Decompiler.ILSpy.Core/Properties/dnSpy.Decompiler.ILSpy.Core.Resources.ru.resx (hunk 2 above)
Comment (geokar2006) [reply Extensions/ILSpy.Decompiler/dnSpy.Decompiler.ILSpy.Core/Properties/dnSpy.Decompiler.ILSpy.Core.Resources.ru.resx:238]: I think your variant is more accurate, I will change to it.

**Review by ElektroKill — APPROVED** [review]

**Pull request merged**